From 47ac44ea8b4f3b1af5d273992f587da50fe8d85e Mon Sep 17 00:00:00 2001
From: Bhushan Shah <bhush94@gmail.com>
Date: Fri, 15 Dec 2017 08:22:43 +0000
Subject: [PATCH] Remove methods which are not upstreamed


--- a/gst/droidcamsrc/gstdroidcamsrcphotography.c
+++ b/gst/droidcamsrc/gstdroidcamsrcphotography.c
@@ -387,24 +387,24 @@
   iface->get_config = gst_droidcamsrc_photography_get_config;
   iface->get_noise_reduction = gst_droidcamsrc_photography_get_noise_reduction;
   iface->set_noise_reduction = gst_droidcamsrc_photography_set_noise_reduction;
-  iface->get_exposure_mode = gst_droidcamsrc_photography_get_exposure_mode;
-  iface->set_exposure_mode = gst_droidcamsrc_photography_set_exposure_mode;
-  iface->get_analog_gain = gst_droidcamsrc_photography_get_analog_gain;
-  iface->set_analog_gain = gst_droidcamsrc_photography_set_analog_gain;
-  iface->get_lens_focus = gst_droidcamsrc_photography_get_lens_focus;
-  iface->set_lens_focus = gst_droidcamsrc_photography_set_lens_focus;
-  iface->get_color_temperature =
-      gst_droidcamsrc_photography_get_color_temperature;
-  iface->set_color_temperature =
-      gst_droidcamsrc_photography_set_color_temperature;
-  iface->get_min_exposure_time =
-      gst_droidcamsrc_photography_get_min_exposure_time;
-  iface->set_min_exposure_time =
-      gst_droidcamsrc_photography_set_min_exposure_time;
-  iface->get_max_exposure_time =
-      gst_droidcamsrc_photography_get_max_exposure_time;
-  iface->set_max_exposure_time =
-      gst_droidcamsrc_photography_set_max_exposure_time;
+  //iface->get_exposure_mode = gst_droidcamsrc_photography_get_exposure_mode;
+  //iface->set_exposure_mode = gst_droidcamsrc_photography_set_exposure_mode;
+  //iface->get_analog_gain = gst_droidcamsrc_photography_get_analog_gain;
+  //iface->set_analog_gain = gst_droidcamsrc_photography_set_analog_gain;
+  //iface->get_lens_focus = gst_droidcamsrc_photography_get_lens_focus;
+  //iface->set_lens_focus = gst_droidcamsrc_photography_set_lens_focus;
+  //iface->get_color_temperature =
+  //    gst_droidcamsrc_photography_get_color_temperature;
+  //iface->set_color_temperature =
+  //    gst_droidcamsrc_photography_set_color_temperature;
+  //iface->get_min_exposure_time =
+  //    gst_droidcamsrc_photography_get_min_exposure_time;
+  //iface->set_min_exposure_time =
+  //    gst_droidcamsrc_photography_set_min_exposure_time;
+  //iface->get_max_exposure_time =
+  //    gst_droidcamsrc_photography_get_max_exposure_time;
+  //iface->set_max_exposure_time =
+  //    gst_droidcamsrc_photography_set_max_exposure_time;
 }
 
 void
